dev-python/h11/h11-0.7.0.ebuild

```diff
@@ -0,0 +1,16 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
Is there a very good reason you're adding two versions? Both given the same visibility, users will just install the newer one and the old one won't be really used or tested.

Reply:
Yes, I am creating all these dev-python/* ebuilds because https://github.com/bilelmoussaoui/Authenticator depends on them and I need that software in my gentoo laptop :) So Authenticator depends on dev-python/wsproto which in turn depends on =dev-python/h11-0.7*). Here is a snippet from my local terminal where I have those packages installed in an overlay:
```
dell /usr/portage/distfiles # equery depends dev-python/h11
 * These packages depend on dev-python/h11:
dev-python/wsproto-0.11.0 (=dev-python/h11-0.7*[python_targets_python2_7(-)?,python_targets_python3_5(-)?,python_targets_python3_6(-)?,-python_single_target_python2_7(-),-python_single_target_python3_5(-),-python_single_target_python3_6(-)])
```

Reply:
My bad... actually it's mitmproxy-3.0.4 who depends on =h11-0.7* through wsproto; not authenticator. I have a few open pull requests against gentoo repo where dependencies to both authenticator and mitmproxy are being added.

Reply:
Ok. Just to confirm: are you sure that those versions are *really* necessary, and it's not just upstream forcing too specific dependencies unnecessarily?

Reply:
To answer your question - no, I am not sure but I think yes, it's a neccesary thing. Here's what I have found in regards to these versions.

mitmproxy-3.0.4 depends on wsproto package. https://github.com/mitmproxy/mitmproxy/blob/v3.0.4/setup.py It states `wsproto>=0.11.0,<0.12.0`

Then wsproto depends on h11 - https://github.com/python-hyper/wsproto/blob/master/setup.py It states `h11 ~= 0.7.0`. 

I quickly scanned the issue queue of wsproto and seems like there are people who have tried wsproto + h11-0.8.* and nothing good came out of it - https://github.com/python-hyper/wsproto/issues/53

Reply:
Let''s wait on https://github.com/python-hyper/wsproto/pull/54